other/password.py

```diff
@@ -62,6 +63,45 @@ def random_characters(chars_incl, i):
     pass  # Put your code here...
 
 
+# This Will Check Whether A Given Password Is Strong Or Not
+# It Follows The Rule that Length Of Password Should Be At Least 8 Characters
+# And At Least 1 Lower, 1 Upper, 1 Number And 1 Special Character
+def strong_password_detector(password: str, min_length: int = 8) -> str:
+    """
+    >>> strong_password_detector('Hwea7$2!')
+    'This is a strong Password'
+
+    >>> strong_password_detector('Sh0r1')
+    'Your Password must be at least 8 characters long'
+
+    >>> strong_password_detector('Hello123')
+    'Password should contain UPPERCASE, lowercase, numbers, special characters'
+
+    >>> strong_password_detector('Hello1238udfhiaf038fajdvjjf!jaiuFhkqi1')
+    'This is a strong Password'
+
+    >>> strong_password_detector(0)
+    'Your Password must be at least 8 characters long'
+    """
+
+    if len(str(password)) < 8:
```

Comment:
```suggestion
    if len(password) < min_length:
```
 Also, short passwords should raise ValueError.  Same with weak passwords.